### ISSUE TITLE
Background correction

### ISSUE THREAD

**Issue by Edison2021** [sprouffske/growthcurver#8]
Hi Kathleen 
Is there any option to skip background correction? Does bg_correct = "none" work for that?

What does " In min(data_n[data_n > 0]) : no non-missing arguments to min; returning Inf" mean?

Thanks
Edison

**Comment by Sarah-Panda** [comment]
Hi! I'm having the same problem, did you get an answer?

**Comment by gl-eb** [comment]
The whole line of code reads `n0_init <- min(data_n[data_n > 0])`. It assumes that background correction / blanking has already been done either by the user or automatically by the growthcurver package. The smallest OD value above zero is used as the initial population size. The `no non-missing arguments to min; returning Inf` error likely arises when all data points for a particular well are below zero, leading to the `data_n > 0` statement returning a vector full of `FALSE`, which in turn returns an empty vector when plugged into `data_n[]`. Finally, the `min()` function can't find a minimum value in an empty vector, leading to the error.

In other words, at least one of the growth curves has resulted in either no growth, or background correction is applied twice (by user and the package), correcting all OD values to below zero. The latter can be fixed by applying blanking that works best for your data and setting the `bg_correct = "none"` option when passing the data to growthcurver.
